README.md

```diff
@@ -48,8 +48,8 @@ node_modules/nw/nwjs/nwjs.app/Contents/MacOS/nwjs .
 # Configuring the Python Environment
 
 In order to run the Python code, a suitable runtime environment is required.
-Python version 3.7 and later are currently supported. It is recommended that a
-Python Virtual Environment is setup for this purpose, rather than using the
+Python Version 3.7 and later are currently supported. It is recommended that a
+Python Virtual Environment is set up for this purpose, rather than using the
```

Comment:
"set up" I agree with. Not sure why you're capitalising version though - it's not part of the name.

Reply:
Oh, sorry! Didn't notice that